src/core/api.c

```diff
@@ -1247,22 +1245,14 @@ MsQuicStreamReceiveComplete(
         !Connection->State.HandleClosed);
 
     if (!Stream->Flags.Started || !Stream->Flags.ReceiveCallPending) {
-        Status = QUIC_STATUS_INVALID_STATE;
```

Comment:
Should we log here so applications which might wonder why they their stream complete call isn't working have a way to find out?